devops/ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Where did this role come from? Was there not an existing role (in Galaxy) we could use or adapt?

If it was adapted from another source (e.g. a blog post), please site that here at top. I'd prefer something battle-tested and community-reviewed.